securedrop/requirements/securedrop-app-code-requirements.txt

```diff
@@ -2,43 +2,209 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --output-file securedrop/requirements/securedrop-app-code-requirements.txt securedrop/requirements/securedrop-app-code-requirements.in
+#    pip-compile --generate-hashes --output-file securedrop/requirements/securedrop-app-code-requirements.txt securedrop/requirements/securedrop-app-code-requirements.in
 #
-alembic==0.9.9
-argon2-cffi==18.1.0
-asn1crypto==0.24.0        # via cryptography
-babel==2.5.1              # via flask-babel
-cffi==1.11.5              # via argon2-cffi, cryptography
-click==6.7                # via flask, rq
-cryptography==2.6.1
-enum34==1.1.6             # via argon2-cffi, cryptography
-flask-assets==0.12
-flask-babel==0.11.2
-flask-sqlalchemy==2.3.2
-flask-wtf==0.14.2
-flask==1.0.2
-ipaddress==1.0.22         # via cryptography
-itsdangerous==0.24        # via flask
-jinja2==2.10.1
-jsmin==2.2.2
-mako==1.0.7               # via alembic
-markupsafe==1.0           # via jinja2, mako
-passlib==1.7.1
-pretty-bad-protocol==3.1.1
-psutil==5.4.3
-pycparser==2.18           # via cffi
-pyotp==2.2.6
-python-dateutil==2.7.2    # via alembic
-python-editor==1.0.3      # via alembic
-pytz==2017.3              # via babel
-qrcode==5.3
-redis==2.10.6
-rq==0.10.0
-scrypt==0.8.0
-sh==1.12.14
-six==1.11.0               # via argon2-cffi, cryptography, python-dateutil, qrcode
-sqlalchemy==1.3.3
-typing==3.6.4
-webassets==0.12.1         # via flask-assets
-werkzeug==0.14.1
-wtforms==2.1              # via flask-wtf
+alembic==0.9.9 \
```

Comment:
Flagging for the purposes of diff review: there are no securedrop-app-code dependency updates as part of this PR